src/python/pants/backend/go/target_types.py

```diff
@@ -74,30 +74,42 @@ class GoModule(Target):
 # -----------------------------------------------------------------------------------------------
 
 
-class GoExtModPackageDependencies(Dependencies):
+class GoExternalPackageDependencies(Dependencies):
     pass
 
 
-class GoExternalModulePath(StringField):
+class GoExternalPackagePathField(StringField):
```

Comment:
I went back and forth between calling it `Package` vs. `Module`. Still not certain I chose right.

Reply:
I would call it "module path" given that was the `go` tool calls it:

Excerpt from `go help mod download`:

```
    type Module struct {
        Path     string // module path
        Version  string // module version
        Error    string // error loading module
        Info     string // absolute path to cached .info file
        GoMod    string // absolute path to cached .mod file
        Zip      string // absolute path to cached .zip file
        Dir      string // absolute path to cached source root directory
        Sum      string // checksum for path, version (as in go.sum)
        GoModSum string // checksum for go.mod (as in go.sum)
    }
```

Note the comments "module path" and "module version"